docgenhtml/features/org.polarsys.capella.docgen.team.feature/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, why dont you just put CDOImageHandler aside the DefaultImageHandler (or merge it into DefaultImageHandler)? (the addon seems doesnt rely on CDO but just on classic sirius api

a new afeature/plugin seems too much for one class that is extended in the same addon.

Reply:
CDOImageHandler won't work be used if there is no cdo:/ uris, and won't be handled if the cdo file handler is not installed
it doesn't explicitly depends on CDO, but it won't work if it isn't here
The explicit dependency was manually added in the cdo feature, so it doesn't install if cdo isn't

Reply:
Removed the feature and extension point